apis/v1alpha1/types.go

```diff
@@ -73,8 +78,7 @@ type ProviderConfigList struct {
 // +kubebuilder:printcolumn:name="CONFIG-NAME",type="string",JSONPath=".providerConfigRef.name"
 // +kubebuilder:printcolumn:name="RESOURCE-KIND",type="string",JSONPath=".resourceRef.kind"
 // +kubebuilder:printcolumn:name="RESOURCE-NAME",type="string",JSONPath=".resourceRef.name"
-// Please replace `PROVIDER-NAME` with your actual provider name, like `aws`, `azure`, `gcp`, `alibaba`
-// +kubebuilder:resource:scope=Cluster,categories={crossplane,provider,PROVIDER-NAME}
+// +kubebuilder:resource:scope=Cluster,categories={crossplane,managed,template}
```

Comment:
I kind of like that `PROVIDER-NAME` is a bit more clear that is needs to be replaced, rather than `template`, which, if you didn't have context, you might think is just a common category we use. What do you think?

Reply:
I didn't really notice it initially only after seeing that `go generate` returns error now with the updated kubebuilder, so I changed it to make it work.

Honestly, that was the first time I noticed it having bootstrapped a few repos using this repo. What I usually do is to look for `MyType`, `template` and similar words and replace them. Having `PROVIDER-NAME` here sort of buries that. In fact, it seems I [didn't notice](https://github.com/crossplane-contrib/provider-tf-aws/blob/main/apis/v1alpha1/types.go#L77) it the last time.

So, I don't know maybe we can change it to `PROVIDER-NAME` everywhere to make find and replace easier, but having both kind of makes you not notice it.

Reply:
Gotcha 👍🏻 I don't feel super strongly -- since you have sued it most recently I'll defer to your judgment :)